dotcom-rendering/src/web/components/Discussion.tsx

```diff
@@ -141,16 +137,6 @@ export const Discussion = ({
 
 	return (
 		<>
-			{commentCount !== undefined && beingHydrated && (
-				<Portal rootId="comment-count-root">
-					<CommentCount
-						isCommentable={isCommentable}
-						commentCount={commentCount}
-						palette={palette}
-					/>
-				</Portal>
-			)}
-
```

Comment:
This is where we remove the old `Portal` - it is replaced with an `Island` instead